src/pages/info/CharacterInfo.tsx

```diff
@@ -92,7 +94,7 @@ export default function CharacterInfo({
                 next={(): Episode[] => episodes}
                 hasMore
                 loader={<h4>{t('loading')}</h4>}
-                height={450}
+                height={height / 3}
```

Comment:
what do you mean "3"?

Reply:
this means that scroll component will have height equals 1/3 of full page height